code/game/machinery/dna_infuser/infuser_entries/infuser_tier_one_entries.dm

```diff
@@ -10,10 +10,10 @@
 	desc = "The guy who said 'Whoever fights monsters should see to it that in the process he does not become a monster' clearly didn't see what a goliath miner can do!"
 	threshold_desc = "you can walk on lava!"
 	qualities = list(
-		"can breath both the station and lavaland air but can't deal with pure O2",
+		"can breath both the station and lavaland air, but be careful around pure 02",
```

Comment:
```suggestion
		"can breath both the station and lavaland air, but be careful around pure oxygen",
```
Wait did you replace this O with a 0?
Frankly it should probably just say the gas name not its atomic composition.

Reply:
What the fuck, uh, my b

Yeah sure no worries I agree with that.